src/main.rs

```diff
@@ -199,6 +199,10 @@ struct Cli {
     #[arg(long, default_value_t = false)]
     only_describe: bool,
 
+    /// Also grab the events from the device through HID-BPF.
+    #[arg(long, default_value_t = false)]
```

Comment:
shouldn't this be a `true/false/auto`? Could re-use `clap::ColorChoice` here which allows for `always/never/auto`